src/openforms/scss/components/admin/_datetime.scss

```diff
@@ -0,0 +1,10 @@
+.datetime-input {
+  display: inline-flex;
+  gap: 3px;
+  padding-right: 2px;
+
+  i {
```

Comment:
we generally avoid targetting HTML elements with selectors. In this case, `.fa` is more appropriate as it indicates it's a Font-Awesome icon.